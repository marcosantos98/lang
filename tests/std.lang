extern "c" puts :: int(str cstr)
extern "c" exit :: int(code int)
