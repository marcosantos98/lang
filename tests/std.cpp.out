/* -- Builtin types -- */
typedef const char* cstr;
/* -- ------------- -- */

extern "C" int puts(cstr);
extern "C" int putchar(int);
extern "C" int exit(int);
void putd(int);

void putd(int d){
if (d>9){
int a = d/10;
d -= 10*a;
putd(a);
}
putchar(48+d);
}

